truss/templates/server.Dockerfile.jinja

```diff
@@ -91,34 +98,24 @@ COPY ./truss /app/truss
 {%- endif %} {#- endif use_local_src #}
 
 COPY ./config.yaml /app/config.yaml
-    {%- if config.live_reload and not config.docker_server %}
+{%- if config.live_reload and not config.docker_server %}
 COPY ./control /control
-# Step 1: Ensure a usable python{{ control_python_version }} is available
-RUN if python{{ control_python_version }} -c 'import venv; venv.EnvBuilder(with_pip=True).create("/tmp/__probe_env")' > /dev/null 2>&1; then \
-      echo "Using system python{{ control_python_version }}"; \
-      python{{ control_python_version }} -m pip install --upgrade pip virtualenv && \
-      python{{ control_python_version }} -m virtualenv /control/.env; \
-    else \
-      echo "Installing Miniforge-based python{{ control_python_version }}..."; \
-      curl -fsSL -o miniforge.sh https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh && \
-      bash miniforge.sh -b -p /opt/conda-control && \
-      rm miniforge.sh && \
-      /opt/conda-control/bin/conda create -y -p /opt/conda-control/envs/py{{ control_python_version }} python={{ control_python_version }} && \
-      /opt/conda-control/envs/py{{ control_python_version }}/bin/pip install --upgrade pip virtualenv && \
-      /opt/conda-control/bin/conda clean -afy && \
-      rm -rf /opt/conda-control/pkgs && \
-      /opt/conda-control/envs/py{{ control_python_version }}/bin/python -m virtualenv /control/.env; \
-    fi
-
-# Step 2: Install requirements into the freshly created venv
+
+# NB(nikhil): Copy the virtual env, and system dependencies for the
+# python version specific to the control server. We need to run ldconfig
+# so dynamic linking works.
+COPY --from=control-server-venv /usr/local/lib/ /usr/local/lib/
```

Comment:
`/usr/local/lib` is very small for the slim image:
```
$ docker run --rm python:3.13-slim ls -la /usr/local/lib/
total 5128
drwxr-xr-x 1 root root     136 Jun  4 17:30 .
drwxr-xr-x 1 root root      10 May 20 00:00 ..
lrwxrwxrwx 1 root root      20 Jun  4 17:30 libpython3.13.so -> libpython3.13.so.1.0
-rwxr-xr-x 1 root root 5175424 Jun  4 17:30 libpython3.13.so.1.0
-rwxr-xr-x 1 root root   67352 Jun  4 17:30 libpython3.so
drwxr-xr-x 1 root root     120 Jun  4 17:30 pkgconfig
drwxr-xr-x 1 root root    3914 Jun  4 17:30 python3.13
```

so I'm not worried about other conflicts